koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -532,7 +536,13 @@ def sigint_handler(*args, **kwargs):
 
         self.logger.info("Response status: %r", response.is_succeeded())
 
-        if response.is_failed():
+        if response.is_cancelled():
+            this_task.cancel()
```

Comment:
Yes, this was an oversight on my part. For some reason I had it in my head that `this_task` was a koji `Task` instance. It is not. I'll get this fixed up.